src/Components/ReviewScore/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReviewStandard } from '@/Types/review';
+import styled from 'styled-components';
+import { StarScore } from './StarScore';
+
+interface ReviewScoreProps {
+  /** 리뷰 평가 기준 */
+  standard: ReviewStandard;
+  /** 평가 점수 */
+  score: number;
+}
+
+export const ReviewScore = ({ standard, score }: ReviewScoreProps) => {
+  return (
+    <ReviewScoreBox>
+      <ReviewScoreRow>
+        <StarScore score={score} />
+        <ReviewScoreText $score={score}>{score}</ReviewScoreText>
+      </ReviewScoreRow>
+      <ReviewStandardText>{standard}</ReviewStandardText>
+    </ReviewScoreBox>
+  );
+};
+
+const ReviewScoreBox = styled.li`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ReviewScoreRow = styled.div`
+  position: relative;
+  width: 48px;
+  height: 48px;
+`;
+
+const ReviewScoreText = styled.p<{ $score: number }>`
+  position: absolute;
+  top: 0;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -40%);
```

Comment:
혹시 `transform`을 따로 적용하신 이유가 있으신지 궁금합니다.
`layout` 과정에서 position으로 위치를 계산한 후 `transform`이 적용되기 때문에 아래처럼 작성해도 될 것 같습니다 🙂

```suggestion
  top: 14px;
  left: calc(50% - 20px);

```

Reply:
세세한 부분까지 확인주셔서 감사합니다!!. 반영하겠습니다 👍